src/services/players/player.hpp

```diff
@@ -71,6 +71,7 @@ namespace big
 		rate_limiter m_invites_rate_limit{10s, 2};
 		rate_limiter m_radio_request_rate_limit{5s, 2};
 		rate_limiter m_radio_station_change_rate_limit{1s, 3};
+		rate_limiter m_explosion_rate_limit{1s, 2};
```

Comment:
We could add more nuance by utilizing our explosion event scanner. Im up for anything you might have in mind.

Reply:
Well rate_limiter is just a basic clock implementation. It doesn't even check if the last explosion was done by the current player you're claiming did it. So if two people set off explosions at once, the 2nd one will get blamed for spamming explosions. A much better implementation will be required. Although I really don't see a proper way to deal with this than what we had before, where the user would just have to sus out modders themselves and block them from spawning explosions if they become excessive.

Reply:
@gir489returns, the rate_limit object exists per player, however it might be that the guy who owns the explosions isn't the same guy that triggered the event (I'm guessing that's what you're talking about?).

Reply:
It does not really matter if the explosion was blamed, the perpetrator will be given a timeout regardless. But im thinking of reversing the limit as in seconds to the actual rate. So a larger time frame with more explosions, this however would cause the initial spam to still be hindering.

Reply:
> @gir489returns, the rate_limit object exists per player, however it might be that the guy who owns the explosions isn't the same guy that triggered the event (I'm guessing that's what you're talking about?).

Yes, I meant that but just poorly articulated it. I get notifications all the time about "X blamed Y for Z explosions!!!!!!" when I know for a fact I literally just put a stickybomb on a car, watched it go near someone and detonated it. The system is far from foolproof.